client/trino-jdbc/pom.xml

```diff
@@ -354,10 +354,6 @@
                                     <pattern>javax.annotation</pattern>
                                     <shadedPattern>${shadeBase}.javax.annotation</shadedPattern>
                                 </relocation>
-                                <relocation>
```

Comment:
where did this go, do we not depend on it anymore?

Reply:
JDBC is not wired using guice. It's just not needed (and I believe it never was)